desktop/ui/src/main/java/org/datacleaner/windows/AbstractDatastoreDialog.java

```diff
@@ -49,6 +52,27 @@
     private final JButton _saveButton;
     private final JButton _cancelButton;
     private final UserPreferences _userPreferences;
+    
+    protected final JXTextField _datastoreNameTextField;
```

Comment:
That's true. Do you think it is a good idea to increase the scope and rework all the subclasses to use inherited name field and its validity check? I can see the file-based dialogs have nice checks, but the rest (Salesforce, SugarCRM, Mongo etc.) do not block the Save button and show a dialog after clicking it.